IntegrationTests/tests_04_performance/test_01_resources/test_1000_autoReadGetAndSet.swift

```diff
@@ -0,0 +1,39 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2021 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIO
+
+func run(identifier: String) {
+    measure(identifier: identifier) {
+        let iterations = 1000
+        let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+        defer {
+            try! group.syncShutdownGracefully()
+        }
+
+        let server = try! ServerBootstrap(group: group)
```

Comment:
this is a total nit but shouldn't we pull out everything apart from what we actually want to measure get/setOptions out of the `measure` block? Then you should get the sync get/set numbers to 0.

Reply:
Ah, duh, of course.